raster/r.horizon/main.c

```diff
@@ -773,7 +773,7 @@ double horizon_height(void)
 {
     double height;
 
-    tanh0 = -1.0 / 0.0;  /* -inf */
+    sscanf("-inf", "%lf", &tanh0); /* -inf */
```

Comment:
r.horizon was significantly refactored, in #3346, #3565 and other PRs, and this code doesn't exist anymore in this shape.